test/images/volume/iscsi/create_block.sh

```diff
@@ -39,9 +39,9 @@ dd if=/dev/zero of=block seek=120 count=1 bs=1M
 mkfs.ext2 block
 
 # Add index.html to it
-mount -o loop block $MNTDIR
-echo "Hello from iSCSI" > $MNTDIR/index.html
-umount $MNTDIR
+mount -o loop block "$MNTDIR"
+echo "Hello from iSCSI" > "$MNTDIR/index.html"
```

Comment:
nit:
```suggestion
echo "Hello from iSCSI" > "${MNTDIR}/index.html"
```
when we're mixing variables and literals we should use braces around the variables

Reply:
YES, thanks.